src/modules/DropManager/DropManager.tsx

```diff
@@ -0,0 +1,174 @@
+}: DropManagerProps) => {
+  const breadcrumbItems = [
+    {
+      name: 'All drops',
+      href: '/drops',
+    },
+    {
+      name: dropName,
+      href: '',
+    },
+  ];
+
+  const getDesktopTableBody = () =>
+    data.map((drop) => (
+      <Tr key={drop.id}>
+        {tableColumns.map((column) => (
+          <Td key={`${column.title}-${drop.id}`}>{column.selector(drop)}</Td>
+        ))}
+        <Td display="flex" justifyContent="right" verticalAlign="middle">
+          <Button mr="1" size="sm" variant="icon" onClick={onCopyClick}>
+            <CopyIcon />
+          </Button>
+          <Button size="sm" variant="icon" onClick={onDeleteClick}>
+            <DeleteIcon color="red" />
+          </Button>
+        </Td>
+      </Tr>
+    ));
+
+  const getMobileTableBody = () =>
+    data.map((drop) => (
+      <Tr key={drop.id}>
+        <Td>
+          <VStack align="flex-start" spacing="2">
+            {tableColumns.map((column) => (
+              <Box key={`${drop.id}-${column.title}`}>{column.selector(drop)}</Box>
+            ))}
+          </VStack>
+        </Td>
+        <Td display="flex" justifyContent="right">
+          <Button mr="1" size="sm" variant="icon" onClick={onCopyClick}>
+            <CopyIcon />
+          </Button>
+          <Button size="sm" variant="icon" onClick={onDeleteClick}>
+            <DeleteIcon color="red" />
+          </Button>
+        </Td>
+      </Tr>
+    ));
+
+  return (
+    <Box px="1" py={{ base: '3.25rem', md: '5rem' }}>
+      <Breadcrumbs items={breadcrumbItems} />
+      <Stack direction={{ base: 'column', md: 'row' }}>
+        {/* Left Section */}
+        <Box flexGrow="1">
+          <Stack
+            direction={{ base: 'column', md: 'row' }}
+            pb={{ base: '4', md: '6' }}
+            pt={{ base: '4', md: '10' }}
+            spacing={{ base: '4', md: '3.125rem' }}
+          >
+            {/* Drop name */}
+            <Stack maxW={{ base: 'full', md: '22.5rem' }}>
+              <Text color="gray.800">Drop name</Text>
+              <Heading>Star Invader 3</Heading>
+            </Stack>
+
+            {/* Drops claimed */}
+            <Stack maxW={{ base: 'full', md: '22.5rem' }}>
+              <Text color="gray.800">{claimedHeaderText}</Text>
+              <Heading>{claimedText}</Heading>
+            </Stack>
+          </Stack>
+          <Text>Track link status and export to them to CSV for use in email campaigns here.</Text>
+        </Box>
+
+        {/* Right Section */}
+        <HStack alignItems="end" justify="center" mt="1rem !important">
+          <Button variant="secondary" w={{ base: '100%', sm: 'initial' }}>
+            Cancel all
+          </Button>
+          <Button variant="secondary" w={{ base: '100%', sm: 'initial' }}>
+            Export .CSV
+          </Button>
+        </HStack>
+      </Stack>
+      <Box>
+        {/* Desktop Table - TODO: Refactor Table component to include mobile version */}
+
+        <Show above="sm">
```

Comment:
i think 'sm' is too small 🤔 , at around ~536px, the table is still showing the desktop size while showing scrollable in X direction
<img width="536" alt="image" src="https://user-images.githubusercontent.com/40631483/210743937-d48c6e0b-f70b-45ed-aec5-cf7085419a24.png">

what if we standardise mobile breakpoints to be `md` instead? I think this will look cleaner 😄

Reply:
yup,  i'm using `md` as breakpoint now